charger/homematic.go

```diff
@@ -39,14 +39,22 @@ func NewCCUFromConfig(other map[string]interface{}) (api.Charger, error) {
 // NewCCU creates a new connection with standbypower for charger
 func NewCCU(embed embed, uri, deviceid, meterid, switchid, user, password string, standbypower float64) (*CCU, error) {
 	conn, err := homematic.NewConnection(uri, deviceid, meterid, switchid, user, password)
+	if err != nil {
+		return nil, err
+	}
 
 	c := &CCU{
 		conn: conn,
 	}
 
+	err = c.conn.Init()
+	if err != nil {
```

Comment:
```suggestion
	if err := c.conn.Init(); err != nil {
```

Dazu den Block nach oben hinter die conn. Wenn das eh gemacht werden muss- warum macht die Connection das nicht gleich selbst beim anlegen? Spart doppelten Code?